clients/privacy-center/packages/fides-consent/README.md

```diff
@@ -85,24 +111,56 @@ if (Fides.consent.data_sales) {
 }
 ```
 
+### Meta Pixel
+
+
+To integrate with [Meta's tracking pixel](https://developers.facebook.com/docs/meta-pixel/get-started),
+call `Fides.meta(options)`, where options is an with the following properties:
+
+- `consent`: boolean - Whether the user consents to [Meta's use of cookies](https://developers.facebook.com/docs/meta-pixel/implementation/gdpr). 
```

Comment:
Facebook's documentation is all very whack-a-mole with various regulations. Their consent grant/revoke specifically refers to GDPR with no explanation of what it actually does.